promptsource/templates/GEM/web_nlg/en/templates.yaml

```diff
@@ -0,0 +1,114 @@
+dataset: GEM/web_nlg
+subset: en
+templates:
+  38342608-5cd7-4ce7-b2e1-905ecd7f4c80: !Template
+    answer_choices: null
+    id: 38342608-5cd7-4ce7-b2e1-905ecd7f4c80
+    jinja: "{{input | join(\"; \")}}\nThe above is a collection of relations. Write\
+      \ descriptive English that contains this information.\n{% for i in references\
+      \ %}\n  ||| {{ i }} \n{% endfor %}"
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
+      original_task: true
+    name: implicit-graph-description
+    reference: ''
+  3e41305c-5461-4cf3-853d-8a6fb5747623: !Template
+    answer_choices: null
+    id: 3e41305c-5461-4cf3-853d-8a6fb5747623
+    jinja: "I will verbalize an abstract representation of a sentence in natural language.\
+      \ To do so, I will first show the representation and then the natural language.\
+      \ The text needs to include all of the information in the representation.\n\n\
+      {{input | join(\", \")}} {% for i in references %}\n  ||| {{ i }} \n{% endfor\
+      \ %}"
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
+      original_task: true
+    name: PALM_prompt
+    reference: used-in-PALM
```

Comment:
how do we know this prompt was used in palm?
all i could find in their paper is the below ..
<img width="817" alt="Screenshot 2022-08-15 at 22 11 45" src="https://user-images.githubusercontent.com/62820084/184710239-17a4d632-de6b-41f4-928b-b12e6120b6f9.png">

Reply:
I believe @sebastianGehrmann shared some NLG prompts for PALM.